tests/test_flow.py

```diff
@@ -2830,6 +2830,62 @@ def test_write_to_parquet_partition_by_datetime(tmpdir):
     assert read_back_df.equals(expected_df)
 
 
+@pytest.mark.parametrize("max_events", [1, 5])
+def test_write_to_single_partition_parquet(tmpdir, max_events):
+    def check_target_parquets(dir, ids=(0, 1, 2)):
```

Comment:
```suggestion
    def check_target_parquets(dir_path, ids=(0, 1, 2)):
```
to avoid shadowing built-in `dir()`.